src/App.js

```diff
@@ -1,55 +1,60 @@
 import React, {useState} from "react";
-import {useTranslation} from "react-i18next";
+import translate from "translate";
 import './App.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Form from 'react-bootstrap/Form';
+import $ from 'jquery';
 
-function App() {
-    const {t, i18n} = useTranslation();
+async function translateTo(lang, text) {
+    translate.engine = "google"; // Or "yandex", "libre", "deepl"
+    translate.key = process.env.GOOGLE_KEY;
+
+    return await translate(text, { from: "en", to: lang });
+}
 
-    const handleChangeLng = (lng) => {
-        i18n.changeLanguage(lng);
-        localStorage.setItem("lng", lng);
-    };
+function App() {
+    const [targetLanguage, setTargetLanguage] = useState("en");
+    const [translation, setTranslation] = useState("");
 
-    const from = document.getElementById("from");
-    let to = "";
+    function handleTranslation(targetLanguage, text) {
+        translateTo(targetLanguage, text).then((result) => {
+            setTranslation(result);
+        }).catch(() => {
+            setTranslation(text);
+        })
+    }
 
     return (
         <Container>
             <Row>
-                <Col xs={5}>
-                    <Form.Select aria-label="Default select example">
+                <Col xs={{ span: 6, offset: 6 }}>
+                    <Form.Select aria-label="Default select example" onChange={(e) => {
+                        setTargetLanguage(e.target.value);
+                        handleTranslation(e.target.value, $("#from").val());
+                    }}>
                         <option value="en">English</option>
-                    </Form.Select>
-                </Col>
-
-                <Col xs={{ span: 5, offset: 2 }}>
-                    <Form.Select aria-label="Default select example" onChange={(e) => handleChangeLng(e.target.value)}>
-                        <option value="es">Spanish</option>
+                        <option value="hy">Armenian</option>
```

Comment:
the better approach is to keep the supported lang list somewhere in config file and to map over them here